### PR TITLE
Dentistry footer added

### LINKED ISSUE #128: Dentistry Footer

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #128 

Test URLs:
- Original: https://www.sunstar-foundation.org/dentistry
- Before: https://main--sunstar-foundation--hlxsites.hlx.live/dentistryfooter
- After: https://dentistry-footer--sunstar-foundation--hlxsites.hlx.live/dentistryfooter

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Block Name    | Documentation
------------- | -------------
 For e.g. cards | [doc-link](https://main--sunstar-foundation--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=0)

- [ ] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------
 For e.g. cards (grid)  | [doc-link](https://main--sunstar-foundation--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=2)

### CODE REVIEW

File: blocks/footer/footer.js [hunk 1]
```diff
@@ -57,9 +85,14 @@ export default async function decorate(block) {
   // fetch footer content
   const footerMeta = getMetadata('footer');
   folder = getMetadata('template');
+  folder = 'dentistry';
```
Comment (jindaliiita) [inline blocks/footer/footer.js:88]: todo remove

**Comment by bosschaert** [comment]
The original URL on sunstar-foundation.org shows nothing. 

Also I see no difference between the Before and After URLs...

Are those URLs correcrt?

**Comment by jindaliiita** [comment]
> The original URL on sunstar-foundation.org shows nothing.
> 
> Also I see no difference between the Before and After URLs...
> 
> Are those URLs correct?

Only Footer is different, you can check the after url footer

**Comment by bosschaert** [comment]
Ah now I see it. Thanks! 👍

**Review by bosschaert — APPROVED** [review]

**Comment by jindaliiita** [comment]
Merging this PR. Thanks @bosschaert for the review.

**Pull request merged**